cardano-crypto-praos/src/Cardano/Crypto/VRF/Praos.hs

```diff
@@ -1,36 +1,23 @@
+{-# LANGUAGE BangPatterns #-}
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE DerivingVia #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE ForeignFunctionInterface #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE ForeignFunctionInterface #-}
-{-# LANGUAGE LambdaCase #-}
 
 -- | Verifiable Random Function (VRF) implemented as FFI wrappers around the
--- implementation in https://github.com/input-output-hk/libsodium
+-- implementation in <https://github.com/input-output-hk/libsodium>
 module Cardano.Crypto.VRF.Praos
   (
   -- * VRFAlgorithm API
     PraosVRF
 
-  -- * Low-level size specifiers
-  --
-  -- Sizes of various value types involved in the VRF calculations. Users of
-  -- this module will not need these, we are only exporting them for unit
-  -- testing purposes.
```

Comment:
so these were not being used in unit tests, despite the comment?

Reply:
Exactly. There are also no real reasons to export them either since there are more natural ways to get to those values: `sizeVerKeyVRF`, `sizeSignKeyVRF`,  `sizeCertVRF`, ` sizeOutputVRF` and `seedSizeVRF`